cli/npm/registry.rs

```diff
@@ -308,13 +308,10 @@ impl NpmRegistryApiInner {
     self
       .load_package_info_from_registry_inner(name)
       .await
-      .with_context(|| {
+      .with_context(||
         format!(
-          "Error getting response at {} for package \"{}\"",
-          self.get_package_url(name),
-          name
-        )
-      })
+          "Error getting response for package \"{name:?}\"",
```

Comment:
```suggestion
          "Error getting response for package '{name}'",
```
There is a debug format here, so the name will go in quotes twice.